smart-contract/contracts/ElectionFactory.sol

```diff
@@ -74,5 +74,10 @@ contract ElectionFactory {
         electionCount = count;
 
         emit CreateElection(count, election, msg.sender, _position);
+        return address(election);
+    }
+
+    function allElections() public view returns(ElectionDetails[] memory){
+        return elections;
     }
```

Comment:
This is made to be paginated and worked on by flowzz. Also make the method external